process/graduation-proposal-template.md

```diff
@@ -1,25 +1,195 @@
 # [Project] Graduation Proposal
 
-_**Insert introduction. See previous proposals for examples. This section should address from a broad perspective why the project feels they are ready to graduation and can state any major accomplishments or milestones.**_
+## Instructions:
+* replace all `???`s with the requested information.
+* elaborate on any prompt to explain the information provided.  Especially if
+  the data provided might seem inconsistent when compared to other projects.
+* for any prompt that requires a private answer put `available upon request`.
+* in most cases, the URLs provided should be to resources within the project's
+  GitHub repo or website rather than personal or private resources.
 
-## Graduation State Criteria
-_**Project should address each graduation criteria listed below**_
+## Project Summary
 
-### * Have committers from at least two organizations.
+1. Name of project: ???
+1. URL to project website:
+   * ???
+1. URL to GitHub repo:
+   * ???
+1. Project description (e.g. what it does, its value, origin, history):
+   * ???
+1. URL to list of project adopters (e.g. ADOPTERS.md or logos on the project 
+   website. For a spec have a list of adopters for the implementation(s) of the
+   spec):
+   * ???
+1. License type (should be Apache 2): ???
+1. URL to license:
+   * ???
+1. TOC sponsor name (if one exists): ???
+1. URL to Core Infrastructure Initiative
+   [Best Practices Badge](https://bestpractices.coreinfrastructure.org/):
+   * ???
+1. URLs to independent and 3rd party security audit results:
+   * ???
+1. URLs to governance documentation:
+   * ???
+1. Names of organizations that have current/active maintainers with the
+   maintainer GitHub IDs (should have at least 2 orgs):
+   * ???org (@???githubID, ...)
+1. URLs to previous CNCF applications (e.g. sandbox, incubator):
+   * sandbox: ???
+   * incubator: ???
 
-### * Have achieved and maintained a [Core Infrastructure Initiative Best Practices Badge](https://bestpractices.coreinfrastructure.org/).
 
-### * Have completed an independent and third party security audit with results published of similar scope and quality as [this example](https://github.com/envoyproxy/envoy#security-audit) which includes all critical vulnerabilities and all critical vulnerabilities need to be addressed before graduation.
+## Graduation Information
```

Comment:
We should also add links to the required reviews, if any are mandatory

Reply:
I modified the security audit bullet to make it so we can add more reviews under there as needed - then they're all in one spot. Are there other reviews I should add?

Reply:
- TAG security was once required (but turned out external audit was enough) - https://github.com/cncf/toc/pull/925#issuecomment-1415993731
- TAG Contributor strategy was required for KEDA - https://github.com/cncf/toc/pull/925#issuecomment-1415993731

Reply:
for TAG Contrib... let's see if we can create the list of requirements that the TAG used and then add them to this doc/PR. I've added a PR with some stuff from KEDA's review - see if anything is missing.